easy_my_coop_loan_website/controllers/main.py

```diff
@@ -0,0 +1,94 @@
+from odoo import http
+from odoo.http import request
+
+from odoo.tools.translate import _
+
+
+class WebsiteLoanIssueSubscription(http.Controller):
+
+    @http.route(['/subscription/get_loan_issue'],
+                type='json',
+                auth="user",
+                methods=['POST'], website=True)
+    def get_loan_issue(self, loan_issue_id, **kw):
+        loan_issue_obj = request.env['loan.issue']
+        if loan_issue_id:
+            loan_issue = loan_issue_obj.sudo().browse(int(loan_issue_id))
+            return {
+                loan_issue.id: {
+                    'max_amount': loan_issue.maximum_amount_per_sub,
+                    'face_value': loan_issue.face_value,
+                    }
+                }
+        else:
+            return False
+
+    @http.route(['/page/loanissuesubscriptionform',

```

Comment:
it'd be great to have a consistent naming with Rémy's pages. Such as

`/struct/<int:struct_id>/loan/subscription`

Reply:
@robinkeunen this is the to be used in stand alone not in the platform so I'm not sure about the the asked form of the route.

Reply:
Well I think it'd be great to have consistent route naming regardless of the use.

Reply:
done